specification/introduction/introduction.tex

```diff
@@ -8,73 +8,69 @@ \section{Design principles}
     The latter imposes severe restrictions on the amount of logic needed to implement the protocol.
 
-    \item[Common high-level functions must be clearly defined] - UAVCAN defines standard services
+    \item[Common high-level functions must be clearly defined] --- UAVCAN defines standard services
     and messages for common high-level functions, such as network discovery, node configuration,
     node software update, node status monitoring (which naturally grows into a vehicle-wide health monitoring),
     network-wide time synchronization, dynamic node ID allocation (a.k.a. plug-and-play node support), etc.
 
-    \item[Open specification and reference implementations] - The UAVCAN specification is open and
+    \item[Open specification and reference implementations] --- The UAVCAN specification is open and
     freely available; the reference implementations are distributed under the terms of the permissive MIT License.
 \end{description}
 
 \section{Capabilities}
 
-This section summarizes the capabilities of the UAVCAN protocol.
-
 UAVCAN-based networks can accommodate up to 127 nodes on the same logical bus.
-More nodes can be added to the system by separating the bus into several independent logical segments
-interconnected via gateway nodes.
 
-UAVCAN supports up to 32768 distinct message data types and up to 256 distinct service data types.
-Part of those are reserved for the standard data types defined by the specification;
-the rest are available for vendor- and application-specific data types.
-More information is provided in the chapter \ref{sec:application_layer}.
+UAVCAN supports up to 65536 message, 256 service, and 8 anonymous data types.
+Some of these identifiers are reserved for the core standard with 
+the rest available for vendor- and application-specific data types.
+More information is provided in chapter \ref{sec:application_layer}.
 
 UAVCAN supports at least\footnote{Depending on the transport protocol.} eight distinct communication priority levels,
 defined in the section \ref{sec:transfer_prioritization}.
-Within each priority level, different types of transfers and different data types are
-prioritized in a well-defined deterministic manner.
+Within each priority level different types of transfers and different data types are
+prioritized in a well-defined, deterministic manner.
 
-The list of transport protocols supported by UAVCAN is provided in the chapter \ref{sec:transport_layer}.
+The list of transport protocols supported by UAVCAN is provided in chapter \ref{sec:transport_layer}.
 Non-redundant, doubly-redundant and triply-redundant transports are supported.
 More information on the physical layer and standardized physical connectivity options
 is provided in the chapter \ref{sec:physical_layer}.
 
+\mbox{\textbf{TODO}: add more about standard application capabilities when this section is complete.} 
```

Comment:
As the document is getting larger, we might be risking forgetting about this TODO. I would rather open a ticket.

Reply:
See #22